pkg/server/server.go

```diff
@@ -494,32 +496,33 @@ func (s *Server) indexHandler(w http.ResponseWriter, r *http.Request) {
-		LoginSuccessURL:       proxy.SingleJoiningSlash(s.BaseURL.String(), AuthLoginSuccessEndpoint),
-		LoginErrorURL:         proxy.SingleJoiningSlash(s.BaseURL.String(), AuthLoginErrorEndpoint),
-		LogoutURL:             proxy.SingleJoiningSlash(s.BaseURL.String(), authLogoutEndpoint),
-		LogoutRedirect:        s.LogoutRedirect.String(),
-		KubeAPIServerURL:      s.KubeAPIServerURL,
-		Branding:              s.Branding,
-		CustomProductName:     s.CustomProductName,
-		StatuspageID:          s.StatuspageID,
-		InactivityTimeout:     s.InactivityTimeout,
-		DocumentationBaseURL:  s.DocumentationBaseURL.String(),
-		AlertManagerPublicURL: s.AlertManagerPublicURL.String(),
-		GrafanaPublicURL:      s.GrafanaPublicURL.String(),
-		PrometheusPublicURL:   s.PrometheusPublicURL.String(),
-		ThanosPublicURL:       s.ThanosPublicURL.String(),
-		GOARCH:                s.GOARCH,
-		GOOS:                  s.GOOS,
-		LoadTestFactor:        s.LoadTestFactor,
-		GraphQLBaseURL:        proxy.SingleJoiningSlash(s.BaseURL.Path, graphQLEndpoint),
-		DevCatalogCategories:  s.DevCatalogCategories,
-		UserSettingsLocation:  s.UserSettingsLocation,
-		ConsolePlugins:        getMapKeys(s.EnabledConsolePlugins),
+		ConsoleVersion:                version.Version,
+		AuthDisabled:                  s.authDisabled(),
+		KubectlClientID:               s.KubectlClientID,
+		BasePath:                      s.BaseURL.Path,
+		LoginURL:                      proxy.SingleJoiningSlash(s.BaseURL.String(), authLoginEndpoint),
+		LoginSuccessURL:               proxy.SingleJoiningSlash(s.BaseURL.String(), AuthLoginSuccessEndpoint),
+		LoginErrorURL:                 proxy.SingleJoiningSlash(s.BaseURL.String(), AuthLoginErrorEndpoint),
+		LogoutURL:                     proxy.SingleJoiningSlash(s.BaseURL.String(), authLogoutEndpoint),
+		LogoutRedirect:                s.LogoutRedirect.String(),
+		KubeAPIServerURL:              s.KubeAPIServerURL,
+		Branding:                      s.Branding,
+		CustomProductName:             s.CustomProductName,
+		StatuspageID:                  s.StatuspageID,
+		InactivityTimeout:             s.InactivityTimeout,
+		DocumentationBaseURL:          s.DocumentationBaseURL.String(),
+		AlertManagerPublicURL:         s.AlertManagerPublicURL.String(),
+		GrafanaPublicURL:              s.GrafanaPublicURL.String(),
+		PrometheusPublicURL:           s.PrometheusPublicURL.String(),
+		ThanosPublicURL:               s.ThanosPublicURL.String(),
+		GOARCH:                        s.GOARCH,
+		GOOS:                          s.GOOS,
+		LoadTestFactor:                s.LoadTestFactor,
+		GraphQLBaseURL:                proxy.SingleJoiningSlash(s.BaseURL.Path, graphQLEndpoint),
+		DevCatalogCategories:          s.DevCatalogCategories,
+		UserSettingsLocation:          s.UserSettingsLocation,
+		ConsolePlugins:                getMapKeys(s.EnabledConsolePlugins),
+		ProjectAccessClusterRolesJSON: s.ProjectAccessClusterRolesJson,
```

Comment:
To simply this we should use the same name here, or?

```suggestion
		ProjectAccessClusterRolesJSON: s.ProjectAccessClusterRolesJSON,
```